bot/cogs/user.py

```diff
@@ -39,31 +40,27 @@ async def get_member(ctx: Context, member: str) -> Member:
         return await MemberConverter().convert(ctx, member)
 
     @staticmethod
-    def get_timedelta(a: datetime.datetime, b: datetime.datetime) -> list:
+    def get_timedelta(a: datetime, b: datetime) -> list:
         """Static method for getting parsed time delta between 2 datetimes."""
         final = []
         delta = relativedelta(a, b)
+
         years = abs(delta.years)
         months = abs(delta.months)
         days = abs(delta.days)
         hours = abs(delta.hours)
         minutes = abs(delta.minutes)
 
-        if minutes and not months and not years:
-            final.append(f'{minutes} minutes')
-
-        if hours:
-            final.append(f'{hours} hours')
-
-        if days:
-            final.append(f'{days} days')
+        dates = {years: "years",
+                 months: "months",
+                 days: "days",
+                 hours: "hours",
+                 months: "months",
+                 minutes: "minutes"}
 
-        if months:
-            final.append(f'{months} months')
-
-        if years:
-            final.append(f'{years} years')
-        final = final[::-1]
+        for date in dates:
+            if date:
+                final.append(f"{date} {dates[date]}")
```

Comment:
You can definitely combine everything here into a single dictionary that gets looped over with `for k, v in dates.items()`.

In the current `dates` dictionary, I'd swap the keys and values so you can have a constant for the keys. All time deltas in the dictionary's new variable (not a variable, but a number that is suseptable to change) can have the absolute function used on them.